.gitpod.yml

```diff
@@ -30,6 +30,14 @@ tasks:
       # Open Moodle site in browser.
       gp ports await 8000 && gp preview $(gp url 8000)
 
+      # Change permissions to allow installation of plugins.
+      bin/moodle-docker-compose exec webserver bash -c 'chmod -R 777 .'
+
+      # Add some data if the file exists.
+      if [ -e "moodle/admin/tool/generator/tests/fixtures/gitpod-basic-scenario.feature" ]; then
```

Comment:
Very small detail, I'd use `-r` (readable) instead of `-e` (or `-f`) that only check for existence.

Reply:
Good suggestion! <3